components/usage-api/usage/v1/usage.proto

```diff
@@ -66,3 +69,16 @@ message ReconcileUsageResponse {
     // ID of the UsageReport generated for this reconcile run.
     string report_id = 2;
 }
+
+message GetCostCenterRequest {
+    string attribution_id = 1;
+}
+
+message GetCostCenterResponse {
+    CostCenter cost_center = 1;
+}
+
+message CostCenter {
+    string attribution_id = 1;
+    int32 spending_limit = 2;
```

Comment:
What's the precision stored in the DB? Is `int32` enough? I'd probably err on the side of caution here and go for int64

Reply:
It's `int(11)` which I understand to be 32 bits?

Reply:
I worry if we set it to `int64` we might truncate something during the conversion? 

I will unhold for now to unblock, but we can discuss again what we should store.